Source/VCGeneration/Checker.cs

```diff
@@ -248,6 +248,10 @@ public bool HasOutput
       get { return hasOutput; }
     }
 
+    /// <summary>
+    /// Gets the amount of time How that the prover spent working. This measures the cost of processing queries,
+    /// but it does not include startup costs (unlike <c>ProverStartTime</c>).
```

Comment:
What startup costs, starting the solver is already done before ProverStateTime, right?

Reply:
It was poor phrasing / a typo.  There's no `ProverStartTime`; it's `VerificationResult.Start` that I meant.